Frameworks/UIKit/UIButton.mm

```diff
@@ -617,6 +618,25 @@ - (void)touchesMoved:(NSSet*)touchSet withEvent:(UIEvent*)event {
     WUXIPointerRoutedEventArgs* routedEvent = [event _touchEvent]->_routedEventArgs;
     [routedEvent setHandled:NO];
 
+    // We're assuming multitouchenabled = NO; it's a button after all.
+    CGPoint point = [[touchSet anyObject] locationInView:self];
+    BOOL currentTouchInside = [self pointInside:point withEvent:event];
+
+    UIControlState newState;
+
+    if (currentTouchInside) {
+        newState = _curState | UIControlStateHighlighted;
+    } else {
+        newState = _curState & ~UIControlStateHighlighted;
+    }
+
+    // Relayout when new state is different than old state
+    if(_curState != newState) {
+        _curState = newState;
+        [self setNeedsDisplay];
+        [self setNeedsLayout];
+    }
```

Comment:
I don't think so - only Button knows whether or not it needs to redisplay/relayout.

---
In reply to: [96943884](https://github.com/Microsoft/WinObjC/pull/1731#discussion_r96943884) [](ancestors = 96943884)